### PR TITLE
release(v1.4.0-alpha.4): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.4.0-alpha.4 release.

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

File: hack/release.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ RELEASE_TOOL_IMAGE="ghcr.io/siderolabs/release-tool:latest"
 
 function release-tool {
   docker pull "${RELEASE_TOOL_IMAGE}" >/dev/null
-  docker run --rm -w /src -v "${PWD}":/src:ro "${RELEASE_TOOL_IMAGE}" -l -d -n ${2} -t "${1}" ./hack/release.toml
+  docker run --net=host --rm -w /src -v "${PWD}":/src:ro "${RELEASE_TOOL_IMAGE}" -l -d -n ${2} -t "${1}" ./hack/release.toml
```
[the remark this answers is not in the document]
Comment (smira) [reply hack/release.sh:9]: it shouldn't hurt in general, but the problem I'm facing is that DNS e.g. for `go.etcd.io` resolves to IPv6, but Docker doesn't do proper IPv6, so seems easier to run with host networking...

**Pull request merged**